app/controllers/trainees/confirm_publish_course_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Trainees
+  class ConfirmPublishCourseController < ApplicationController
+    before_action :authorize_trainee
+
+    def edit
+      render body: "ye olde confirm page"
```

Comment:
Still needed?

Reply:
Its going to be replaced in this PR https://github.com/DFE-Digital/register-trainee-teachers/pull/738 that goes on top of this. I had originally just added this so I could manually test without it erroring but I can remove